src/__tests__/use-global-directive.js

```diff
@@ -0,0 +1,18 @@
+import '@testing-library/jest-dom/extend-expect'
+import {render, wait} from '@testing-library/vue'
+import {value as valueDirective} from './components/directives/v-value'
+import CompWithDirective from './components/CompWithDirective.vue'
+
+test('element should have the same value as the directive', async () => {
```

Comment:
I feel the example is a bit "wordy" (it says `value` a lot 😝).

I think it would be best if we used a simpler example with a clearer directive. Also I'd love to avoid using `updateProps`, as it has actually nothing to do with directives, and people may get the wrong impression.

What do you think about that?